server/utils/crmDisplayHelper.ts

```diff
@@ -1,12 +1,75 @@
 import { CrmResponse } from '@eqApi'
-import { Section, ShowOrHideWhen } from '@crmDisplay'
+import { Section, ShowOrHideWhen, Subsection } from '@crmDisplay'
 import _ from 'lodash'
 
 export const includeSection = <T extends CrmResponse>(section: Section, crmResponse: T): boolean => {
   if (section.hideWhen && conditionsMet(section.hideWhen, crmResponse)) {
     return false
   }
-  return !section.showWhen || conditionsMet(section.showWhen, crmResponse)
+
+  const include = !section.showWhen || conditionsMet(section.showWhen, crmResponse)
+
+  if (include && isSectionEmpty(section, crmResponse)) {
```

Comment:
Add a test case for when section is empty under the   describe('includeSection()' ... in crmDisplayHelper.test.ts